### PR TITLE
Video | Demo | Add demo of video triggering modal

### PR DESCRIPTION
**Pull request by danielamorse**
## Jira

http://vjira2:8080/browse/BDS-1602

## Summary

Add demo of video triggering a modal to open on play.

## Details

Drupal implementation works better if video triggers modal to open, rather than modal triggering video to play. Add demo showing how this is done in Bolt.

## How to test

1. Run code locally
2. Go to [Video with Modal](http://localhost:3000/pattern-lab/patterns/02-components-video-70-video-with-modal/02-components-video-70-video-with-modal.html) page and verify that clicking on the "Play the video" button opens the plays the video in a modal and closing pauses the video.

### CODE REVIEW

File: docs-site/src/pages/pattern-lab/_patterns/02-components/modal/45-modal-usage-video.twig [hunk 1]
```diff
@@ -14,7 +14,7 @@
 </script>
 {% endset %}
 
-<bolt-text headline font-size="xlarge">Video Modal Usage</bolt-text>
+<bolt-text headline font-size="xlarge">Video Modal Usage: Modal Triggers Video</bolt-text>
 <bolt-text>Custom javascript is needed to play the video when the modal opens and pause the video the modal closes.</bolt-text>
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/02-components/modal/45-modal-usage-video.twig]: Missing `when` before `the modal closes`.

File: docs-site/src/pages/pattern-lab/_patterns/02-components/video/70-video-with-modal.twig [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{% set javascript %}
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/02-components/video/70-video-with-modal.twig]: Having the exact same page under 2 different folders is not ideal, one must remember to update both pages. Is there a better way to handle this? Simply write some copy on this page and redirect user to the modal demo page with a link? Thoughts? @danielamorse @sghoweri

**Comment by danielamorse** [comment]
@mikemai2awesome I combined the two demos on the same page as we discussed.

File: docs-site/src/pages/pattern-lab/_patterns/02-components/video/70-video-with-modal.twig (hunk 2 above)
Comment (sghoweri) [reply docs-site/src/pages/pattern-lab/_patterns/02-components/video/70-video-with-modal.twig]: FWIW, I'm fine if we have occasionally have duplication for these sorts of things, especially if it means the demos are easier to follow from an end user (ie. source code shows what it actually needs to show).

There's a ton of work to be done to reduce the total # of duplication / baked in components in our PL demos anyways (v3.0 cleanup) so this sort of detail isn't as mission critical to address, especially if there's already a lot of other high priority work already in the queue.

#gotToPickOurBattles

**Pull request merged**